spec/registry.md

```diff
@@ -381,6 +381,69 @@ together with the resolved options' values.
 
 The _function_ `:integer` performs selection as described in [Number Selection](#number-selection) below.
 
+### The `:math` function
+
+The function `:math` is a selector and formatter for matching or formatting
```

Comment:
It's not a formatting function, is it? A formatting function would require all of `:number` or `:integer`'s options, e.g. to control zero digits, separators, and the like.

Reply:
As proposed, you would be able to format a `:math` annotated value, so doesn't that make it a formatting function? If you need the options, you can set them in a separate declaration.